pkgs/development/tools/haskell/dconf2nix/default.nix

```
@@ -0,0 +1,21 @@
+{ mkDerivation, base, containers, optparse-applicative, parsec
+, stdenv, text
+}:
+mkDerivation rec {
+  pname   = "dconf2nix";
+  version = "0.0.5";
```

linting, sorry for the nits
```suggestion
  version = "0.0.5";

```